extensions/smallrye-openapi/deployment/src/main/java/io/quarkus/smallrye/openapi/deployment/SmallRyeOpenApiProcessor.java

```diff
@@ -509,6 +595,14 @@ private Result findStaticModel() {
     private OpenApiDocument loadDocument(OpenAPI staticModel, OpenAPI annotationModel,
             List<AddToOpenAPIDefinitionBuildItem> openAPIBuildItems) {
         OpenApiDocument document = prepareOpenApiDocument(staticModel, annotationModel, openAPIBuildItems);
+
+        Config c = ConfigProvider.getConfig();
+        String title = c.getOptionalValue("quarkus.application.name", String.class).orElse("Generated");
+        String version = c.getOptionalValue("quarkus.application.version", String.class).orElse("1.0");
```

Comment:
Not important at all, but elsewhere we use `<<unset>>`

Reply:
What is `<<unset>>` ? You mean in the orElse ? I kept the orElse as it was previously. This only apply in tests where the application name is not set. This makes the test cases pass.

Reply:
Yeah, I meant for the `orElse`. But as I mentioned, it doesn't matter at all

Reply:
Ok, I am keeping it as it, to make the tests pass without having to change them :)

Reply:
👍🏼 . Sounds good